### PR TITLE
update execute script to return the string result and added scrollToP…

### PR DESCRIPTION
**Pull request by refael-m**
as part of [APPS-6231](https://cloudinary.atlassian.net/browse/APPS-6231)

### CODE REVIEW

File: src/commons/BrowserUtils.ts [hunk 1]
```diff
@@ -495,6 +495,19 @@ export namespace BrowserUtils {
...
   }
 
+  /**
+   * Scrolls the element to a particular place.
+   * @param selector element selector
+   * @param x is the pixel along the horizontal axis of the element.
+   * @param y is the pixel along the vertical axis of the element.
+   * @private
+   */
+  export function scrollToPixels(selector: string, x: number, y: number): void {
```
Comment (FelixZilber) [inline src/commons/BrowserUtils.ts:505]: please add tests

File: src/commons/BrowserUtils.ts [hunk 2]
```diff
@@ -42,9 +42,9 @@ export namespace BrowserUtils {
    * for execution in the context of the currently selected frame
    * @param script - js script to execute
    */
-  export function executeScript(script: string): void {
+  export function executeScript(script: string): string {
     Reporter.debug(`Executing script: '${script}'`);
-    tryBlock(() => browser.execute(script), `Failed to execute script: ${script}`);
+    return tryBlock(() => browser.execute(script), `Failed to execute script: ${script}`);
```
Comment (FelixZilber) [inline src/commons/BrowserUtils.ts:47]: add test for this change

**Review by FelixZilber — CHANGES_REQUESTED** [review]

File: src/commons/BrowserUtils.ts (hunk 1 above)
Comment (refael-m) [reply src/commons/BrowserUtils.ts:505]: added

File: src/commons/BrowserUtils.ts (hunk 2 above)
Comment (refael-m) [reply src/commons/BrowserUtils.ts:47]: added

File: src/test/specs/ScrollToPixels.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { assert } from 'chai';
+import { BrowserUtils } from '../..';
+import { describeCommon } from '../TestHelper';
+
+const TEST_DIV_ID: string = 'scrollToPixelsDiv';
+const TEST_FIELD_SELECTOR: string = `//*[@id='${TEST_DIV_ID}']`;
+
+describeCommon('ScrollToPixels', () => {
```
Comment (FelixZilber) [inline src/test/specs/ScrollToPixels.ts:8]: the file name must end with 'Spec.ts'

**Review by FelixZilber — CHANGES_REQUESTED** [review]

File: src/test/specs/ScrollToPixels.ts (hunk 3 above)
Comment (refael-m) [reply src/test/specs/ScrollToPixels.ts]: done

File: src/commons/BrowserUtils.ts (hunk 1 above)
Comment (FelixZilber) [inline src/commons/BrowserUtils.ts:505]: suggest changing the name to "scrollTo"

**Review by FelixZilber — CHANGES_REQUESTED** [review]

**Review by FelixZilber — APPROVED** [review]

**Pull request merged**